source/java-drivers.txt

```diff
@@ -7,11 +7,11 @@ MongoDB Java Drivers
 .. default-domain:: mongodb
 
 .. facet::
-  :type: programming_language
+  :name: programming_language
```

Comment:
Suggestion:
I think these should be indented one more space, similar to the facet attributes in the pymongo.txt file in this PR.